tests/cases/fourslash/getOccurrencesSuper3.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference path='fourslash.ts' />
+
+////let x = {
+////    a() {
+////        return [|s/**/uper|].b();

```

Comment:
currently type of `super` is `any` however i'd say that we should consider to use more elaborate type for both `this` and `super` in methods of object literal expressions (@rbuckton and myself have seen cases when this will be very useful during Migrate-a-thon)